packages/cubejs-query-orchestrator/src/orchestrator/RedisPool.ts

```diff
@@ -1,23 +1,22 @@
+import Redis from 'ioredis';
 import genericPool, { Pool, Options as PoolOptions } from 'generic-pool';
+import { createRedisClient } from './RedisFactory';
 
-import { createRedisClient, AsyncRedisClient } from './RedisFactory';
-
-export type CreateRedisClientFn = () => PromiseLike<AsyncRedisClient>;
-
+export type CreateRedisClientFn = () => PromiseLike<Redis.Redis>;
 export interface RedisPoolOptions {
   poolMin?: number;
   poolMax?: number;
   createClient?: CreateRedisClientFn;
-  destroyClient?: (client: AsyncRedisClient) => PromiseLike<void>;
+  destroyClient?: (client: Redis.Redis) => PromiseLike<void>;
```

Comment:
To use it like `Redis`, instead of `Redis.Redis`

```suggestion
  destroyClient?: (client: Redis) => PromiseLike<void>;
```